Compiling without lua broken

- Program: Authoritative
 - Issue type: Bug report

### Short description

```
./configure --with-modules="" --without-lua --enable-unit-tests
make -j8
```

leads to:

```
lua-auth4.o:(.data.rel.ro._ZTV8AuthLua4[_ZTV8AuthLua4]+0x20): undefined reference to `AuthLua4::postPrepareContext()'
lua-auth4.o:(.data.rel.ro._ZTV8AuthLua4[_ZTV8AuthLua4]+0x28): undefined reference to `AuthLua4::postLoad()'
```

### Description

The vtable for AuthLua4 misses the promised `virtual` `override` functions from the header if lua is disabled:
```
  virtual void postPrepareContext() override;
  virtual void postLoad() override;
```

--- a/pdns/lua-auth4.cc
+++ b/pdns/lua-auth4.cc
@@ -17,6 +17,14 @@ DNSPacket *AuthLua4::prequery(DNSPacket *q) { return NULL; }
 
 AuthLua4::~AuthLua4() { }
 
+void AuthLua4::postPrepareContext()
+{
+}
+
+void AuthLua4::postLoad()
+{
+}
+
 #else
 
 #undef L

